pkannan3/guide-me
# TripsCards

Updates:

1. NavBar is sticky and moves as the user scrolls
2. Added Image placeholder for trip cards
3. Changed card and card content size
4. Changed button colors, icons, and applied "transform" property
5. Add a "View" button
6. Cards now wrap properly
7. Adjusted "+" card size

Test:

1. Create a minimum of 3 trips.
2. Confirm that the Edit and Delete button icons are updated.
3. Select any one of the buttons (View, Edit, or Delete). This should create a border indicating this card is currently selected. It should also activate all the tabs on the NavBar.
4. Collapse the window to confirm the cards wrap properly.
5. While collapsed scroll to confirm the NavBar moves with the scroll.
